File: src/sequence/world/meta/SqStat.java
package sequence.world.meta;

import mindustry.world.meta.Stat;
import mindustry.world.meta.StatCat;

public class SqStat {
    public static final StatCat sequence = new StatCat("sequence");

    public static final Stat sequenceOrder = new Stat("sequenceOrder", sequence),

    sequenceEffect = new Stat("sequenceEffect", sequence),

    informationAnnihilationFactor = new Stat("informationAnnihilationFactor");
}

File: src/sequence/world/meta/SqStat.kt
package sequence.world.meta

import mindustry.world.meta.Stat
import mindustry.world.meta.StatCat

object SqStat {
    val sequence = StatCat("sequence")
    val sequenceOrder = Stat("sequenceOrder", sequence)
    val sequenceEffect = Stat("sequenceEffect", sequence)
    val informationAnnihilationFactor = Stat("informationAnnihilationFactor")
    val ammoOverride = Stat("ammo", StatCat.function)
    val reloadOverride = Stat("reload", StatCat.function)
}
